vcd/resource_vcd_vapp_vm.go

```diff
@@ -88,6 +92,8 @@ func resourceVcdVAppVm() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
+				ValidateFunc: anyValueWarningValidator(true,
+					"With next version `name` as computer name won't be set together with `initscript`. Please use `computer_name`."),
```

Comment:
Which version are we talking about?
It is unclear what behavior we are promising now as opposed to later

Reply:
We need dedication, I could change to 2.6, but would need @lvirbalas confirmation

Reply:
Please define what is the current behavior and what is the promised one. From this description, it is unclear.

Reply:
I believe we should be printing the warning from within the function call in that very specific use case (I just posted a comment in that part), because printing it on initscript means spamming the users who are using initscript but who don't care about computer name.

Reply:
In function warning is workaround/hack which we use when I found for upload functionality. Yes we can, but downside you will not see such error on `plan` phase - only on `execution`. P.s. in general if add warning in function - we only avoid showing warning when `computer_name` is set, but otherwise always also will be showed.